tests/unit/metadata_manager_test.py

```diff
@@ -99,20 +96,25 @@ def test_cache_data(metadata_manager: MetadataManager):
 """
 
 
+@pytest.mark.skipif(
```

Comment:
Some of the `MetaDataManager` tests still need some love.

For this one it is not clear what it actually is supposed to test exactly.